Aquality.Selenium.Template/Aquality.Selenium.Template/Enums/ContactUsTextFields.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace Aquality.Selenium.Template.Enums
+{
+    public enum ContactUsTextFields
+    {
+        [Description("Please fill in the required field.")]
```

Comment:
could we use constant for duplicated descriptions?

Reply:
Fixed